.coderabbit.yaml

```
@@ -0,0 +1,33 @@
+        ignore_title_keywords:
+            - "WIP"
+            - "DO NOT MERGE"
+        drafts: false
+        base_branches:
+            - "develop"
+            - "feature/.*"
+            - "!master"
+            - "!hotfix/.*"
+            - "!release/.*"
+chat:
+    auto_reply: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)